README.rst

```diff
@@ -8,6 +8,12 @@ TraitsUI: Traits-capable windowing framework
 .. image:: https://ci.appveyor.com/api/projects/status/n2qy8kcwh8ibi9g3/branch/master?svg=true
    :target: https://ci.appveyor.com/project/EnthoughtOSS/traitsui/branch/master
 
+.. image:: https://github.com/enthought/traitsui/workflows/ETS%20from%20source/badge.svg 
+.. image:: https://github.com/enthought/traitsui/workflows/Test%20with%20PyPI/badge.svg
+.. image:: https://github.com/enthought/traitsui/workflows/Test%20with%20EDM/badge.svg
```

Comment:
This 'Test with EDM' is now gone.

Reply:
More accurately... it has not been put up officially with master. It was run from a development branch. 
This makes me realize that we might want to add branch filter to make sure these badges the status for the master branch: https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-the-branch-parameter
We can see that the "Test with EDM" for master has no status: https://github.com/enthought/traitsui/workflows/Test%20with%20EDM/badge.svg?branch=master

Note that some don't like the PyPI page showing the build status of the master branch. For me, I don't mind either way but this might be a question for @corranwebster.